### PR TITLE
Include link to csharp REPL

### PR DESCRIPTION
**Pull request by binary10**
In Miscellaneous tools section.

### CODE REVIEW

File: docs/tools+libraries/tools/index.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ Miscellaneous tools
 -   sqlsharp: command line SQL client for most of the the System.Data based DB Connectors
 -   [Gendarme](/docs/tools+libraries/tools/gendarme/): Rule-based assembly analyser for developers.
 -   [Monoxide](/archived/monoxide): Interactive assembly analyser.
+-   [csharp](/repl): An interactive shell (REPL) for C#.
```
Comment (akoeplinger) [inline docs/tools+libraries/tools/index.md]: Did you plan to include a page for the repl? If not, please remove the brackets for the markdown link.

**Comment by binary10** [comment]
I fixed the link and tested it. Works for me.

**Comment by akoeplinger** [comment]
Thanks, I didn't realize we already had a page for the repl. The link needs to be relative to the site root, I fixed it and merged.

**Comment by binary10** [comment]
Thanks for correcting it for the website and for pulling in the change.

**Pull request merged**